google/cloud/security/inventory/transform_util.py

```diff
@@ -39,6 +39,16 @@ def flatten_projects(projects):
     for project in (project for d in projects \
                     for project in d.get('projects', [])):
         project_json = json.dumps(project)
+
+        project_id = project.get('projectId')
+        lifecycle_state = project.get('lifecycleState')
+        parent = project.get('parent', {})
+
+        if lifecycle_state != 'ACTIVE':
```

Comment:
qq: Instead of filtering here, would we want to do the filter when we actually make the crm api call?  i.e. only retrieve active projects?  Is there any reason why we would want to retrieve non-active projects?
https://github.com/GoogleCloudPlatform/forseti-security/blob/master/google/cloud/security/common/gcp_api/cloud_resource_manager.py#L86